### PR TITLE
Added Cypress E2E folder and basic configurations

### PR DESCRIPTION
**Pull request by Jonsy13**
Signed-off-by: Vedant <vedant.shrotria@mayadata.io>

- **Issue:**
fixes: https://github.com/litmuschaos/litmus/issues/1702 and https://github.com/litmuschaos/litmus/issues/1688

**This PR is for:** 
- Adding some litmus portal tests like accessibility to login and workflow

### CODE REVIEW

File: CypressE2E/cypress/support/commands.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
```
Comment (uditgaurav) [inline CypressE2E/cypress/support/commands.js:25]: Why do we comment it?

**Comment by uditgaurav** [comment]
@Jonsy13 Do you Want to put the output of the test as well?

File: CypressE2E/cypress/support/commands.js (hunk 1 above)
Comment (Jonsy13) [reply CypressE2E/cypress/support/commands.js:25]: This file is used for making our custom functions.I just left it like this, if someone wants to add any function they can refer to these.Right now, i'm not using any custom function.

File: CypressE2E/cypress/integration/Login/login.spec.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/// <reference types="Cypress" />
+
+describe("Testing the Rechablity of Login page",()=>{
```
Comment (uditgaurav) [inline CypressE2E/cypress/integration/Login/login.spec.js:3]: Testing the accessibility of Login page

File: CypressE2E/cypress/integration/Workflow/workflow.spec.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+/// <reference types="Cypress" />
+
+describe("Testing the Rechablity of Workflow page",()=>{
```
Comment (uditgaurav) [inline CypressE2E/cypress/integration/Workflow/workflow.spec.js:3]: Testing the accessibility of Workflow page

File: CypressE2E/cypress/integration/Login/login.spec.js (hunk 2 above)
Comment (Jonsy13) [reply CypressE2E/cypress/integration/Login/login.spec.js:3]: Yes, I'm changing these.

**Comment by uditgaurav** [comment]
Do you also want to add it in gitlab-ci.yml file?

**Comment by Jonsy13** [comment]
Output

**Comment by uditgaurav** [comment]
Do you also want to add this test to our gitlab-ci.yml ?

**Comment by Jonsy13** [comment]
Yes,i want to add these tests in gitlab-ci.yaml.

**Pull request merged**